v2_papers/2016/ITC_flowStatistics.json

```diff
@@ -0,0 +1,307 @@
+        "types": [],
+        "generation": "synthetic",
+        "generation_year": "missing",
+        "covered_period": "missing",
+        "details": [
+          "raw"
+        ],
+        "subsets": [],
+        "anonymized": true
+      }
+    ]
+  },
+  "preprocessing": {
+    "performed_feature_selection": false,
+    "packet_analysis_oriented": false,
+    "flow_analysis_oriented": true,
+    "flow_aggregation_analysis_oriented": false,
+    "tools": [
+      {
+        "name": "own_java_scripts",
+        "detail": "used to build the framework",
+        "availability": "private"
+      },
+      {
+        "name": "jpcap",
+        "detail": "java library to parse packets",
+        "availability": "private"
+      }
+    ],
+    "normalization_type": "missing",
+    "transformations": [
+      "flow_extraction"
+    ],
+    "final_data_format": "numerical_vectors",
+    "flows": [
+      {
+        "selection": "expert_knowledge",
+        "role": "main",
+        "main_goal": "traffic_classification",
+        "active_timeout": "missing",
+        "idle_timeout": "missing",
+        "bidirectional": false,
+        "features": [
+          "packetTotalCount",
+          "flowDurationMilliseconds",
+          "octetTotalCount",
+          {
+            "add": [
+              {
+                "apply": [
+                  "octetTotalCount",
+                  {
+                    "select_slice": [
+                      0,
+                      10
+                    ]
+                  }
+                ]
+              }
+            ]
+          },
+          {
+            "minimum": [
+              "ipTotalLength"
+            ]
+          },
+          {
+            "maximum": [
+              "ipTotalLength"
+            ]
+          },
+          {
+            "mean": [
+              "ipTotalLength"
+            ]
+          },
+          {
+            "stdev": [
+              "ipTotalLength"
+            ]
+          },
+          {
+            "minimum": [
+              "_interPacketTimeMicroseconds"
+            ]
+          },
+          {
+            "maximum": [
+              "_interPacketTimeMicroseconds"
+            ]
+          },
+          {
+            "mean": [
+              "_interPacketTimeMicroseconds"
+            ]
+          },
+          {
+            "stdev": [
+              "_interPacketTimeMicroseconds"
+            ]
+          }
+        ],
+        "key_features": [
+          "sourceIPv4Address",
+          "destinationIPv4Address",
+          "sourceTransportPort",
+          "destinationTransportPort",
+          "protocolIdentifier"
+        ]
+      },
+      {
+        "selection": "expert_knowledge",
+        "role": "main",
+        "main_goal": "traffic_classification",
+        "active_timeout": "missing",
+        "idle_timeout": "missing",
+        "bidirectional": true,
+        "features": [
+          "flowDurationMilliseconds",
+          "first10senders",
```

Comment:
This is not a IANA feature, so it should start with double underscore (``__``), unless you think the feature is useful for other papers, in which case it should start with a single underscore (``_``) and added to our list of valid features.
Check details on these rules here: https://nta-meta-analysis.readthedocs.io/en/latest/features.html